vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'cran', title: 'CRAN/METACRAN' });
+module.exports = t;
```

Comment:
The usual style I see is to put `module.exports` at the very bottom of the file.